tests/test_data.py

```diff
@@ -576,15 +576,15 @@ def test_Kirk19_ingest(db):
         )
         .astropy()
     )
-    assert len(t) == 290, f"found {len(t)} photometry entries for {telescope}"
```

Comment:
Above test is incorrect. It should be just for telescope and not ref.  Total number should be 1007 *before* your ingest.